handler.js

```diff
@@ -764,6 +777,54 @@ const getInformationAboutWellnessActivities = async function(
   }
 };
 
+
+/**
+ * Handle the intent request of someone making a noise alert report
+ * @param {Object} intentRequest Intent request information
+ * @param {function} callback Callback function to handle the response
+ */
+const reportNoiseAlert = async function(intentRequest, callback) {
+  const floor = intentRequest.currentIntent.slots.floor;
+
+  // Retrieve user name from Slack
+  const userId = getSlackUserId(intentRequest);
+  let userInfo = null;
+  try {
+    // Retrieve user information from Slack using the SDK
+    userInfo = await getSlackUserInfo(userId);
+  } catch (error) {
+    console.log(error);
+    fulfillWithError(
+      intentRequest,
+      callback,
+      "Failed to retrieve user's info from Slack"
+    );
+    return;
+  }
+
+  const content = {
+    reporter: userInfo.user.name,
+    location: {
+      floor: floor
+    }
+  };
+
+  console.log(content);
+
+  // Update the plate number using ParkingBot API
+  noiseAlertAPI
+    .post('/noisereport', content)
+    .then(res => {
+      console.log(res.data);
+      const message = `Ok. Your request was sent.`;
+      fulfillWithSuccess(intentRequest, callback, message);
+    })
+    .catch(error => {
+      console.log(error);
+      fulfillWithError(intentRequest, callback, error.response.data.error);
+    });
```

Comment:
Since you're already in an `async` function, you can use `await` instead of promise-based then

refactor to:
```javascript
try {
  const res = await noiseAlertAPI.post('/noisereport', content);
   // success
} catch {
  // error handling
}
```

Reply:
Good catch. I just copied and pasted the same structure from other handlers as I was requested to do this fast so the other guys working on the noise alert project could advance.  I will updated as suggested though. Perhaps we should extend this to other handlers as part of the refactor you worked on: https://github.com/gorillalogic/winston-lambda/issues/6

We still need to push for your AWS access and permissions so we can proceed with your PR.

Reply:
Yup!, that was part of the refactor too, I refactored a lot of promises to async/await, yet we still need to test for all of the possible scenarios.